functorch/_src/aot_autograd.py

```diff
@@ -387,6 +388,12 @@ def fake_fn(primals, tangents):
     with torch.no_grad():
         with track_graph_compiling("joint"):
             fw_module, bw_module = aot_config.partition_fn(fx_g, joint_inputs)
+            fw_outs = [n for n in fw_module.graph.nodes if n.op == "output"][0].args[0]
+            # we only need to bookkeep the symints that are saved for bw, not any symints
+            # the user forward might have returned in its own output
+            fw_outs = fw_outs[_num_outs:]
+            symint_outs = [n for n in fw_outs if is_sym_node(n)]
```

Comment:
If the symints are guaranteed to be suffix, we can assert that this is the case here.

Reply:
TODO assert this.